docs/pages/inboxes/build-inbox.md

```diff
@@ -745,6 +745,92 @@ let orderFilteredConversations = try await client.conversations.listGroups(conse
 
 :::
 
+## Retrieve a wallet address from an inbox ID
+
+An XMTP inbox ID can have an array of addresses associated with it. To retrieve an address from an inbox ID, use the `inboxState` method. This retrieves the current state of the inbox, which includes the primary address associated with it.
+
+### Best practices
+
+When displaying an address for an inbox ID, consider these best practices:
+	
+- For 1:1 chats: Display the primary address or name associated with the inbox ID.
+- For group chats: Display the name of the group, or enable users to choose the one address they want to display from the associated addresses.
```

Comment:
Link to this doc to show how to get a group chat name? https://docs.xmtp.org/inboxes/group-metadata#get-a-group-chat-name